pkgs/top-level/all-packages.nix

```diff
@@ -29287,6 +29287,8 @@ with pkgs;
 
   caladea = callPackage ../data/fonts/caladea { };
 
+  candy-icons = callPackage ../data/icons/candy-icons { };
```

Comment:
Use the new [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format so you don't have to edit this file

Reply:
Thank you for your review! I have made all changes.